traffic_ops/client/cachegroup.go

```diff
@@ -122,8 +123,8 @@ func (to *Session) GetCacheGroupsNullable() ([]tc.CacheGroupNullable, ReqInf, er
 
 // GET a CacheGroup by the CacheGroup ID.
 func (to *Session) GetCacheGroupNullableByID(id int) ([]tc.CacheGroupNullable, ReqInf, error) {
-	route := fmt.Sprintf("%s/%d", API_CACHEGROUPS, id)
-	resp, remoteAddr, err := to.request(http.MethodGet, route, nil)
+	url := fmt.Sprintf("%s?id=%v", API_CACHEGROUPS, id)
+	resp, remoteAddr, err := to.request(http.MethodGet, url, nil)
```

Comment:
This is fine, but I'd like to point out that this really is just a request path, not a full URL.